.schema/devbox.schema.json

```diff
@@ -45,6 +37,12 @@
                                 "excluded_platform": {},
                                 "glibc_patch": {}
                             }
+                        },
```

Comment:
i think lines 36 and 37 should be `platforms` and `excluded_platforms` with `s` for plural, and their type should be array-of-strings

Reply:
Gotcha, will update. Is there a list of valid entries for the platform? I could make this an enum so that the autocomplete is more useful.

Reply:
oh yeah, good call, can be an enum. Here ya go:
```
❯ devbox add hello --platform=foo
Info: Adding package "hello@latest" to devbox.json

Error: Unsupported platform: foo. Valid platforms are: [aarch64-darwin aarch64-linux i686-linux x86_64-darwin x86_64-linux armv7l-linux]
```